<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmReceipt</class>
 <widget class="QMainWindow" name="frmReceipt">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Receipt viewer | SmartClass</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/logos/Logo.png</normaloff>:/images/logos/Logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#titleBar{
	background-color: #FFF;
	border-top: 1px solid orange;
	border-left: 1px solid orange;
	border-right: 1px solid orange;
	color: #000;
}

QWidget#centralWidget, QStatusBar{
	background-color: #FFF;
	border-left: 1px solid orange;
	border-right: 1px solid orange;
}

QStatusBar{
	border-bottom: 1px solid orange;
}

/*QTabWidget::pane {
    border-top: 1px solid orange;
}*/

QSlider::handle:horizontal {
    background: orange;
}

/* Vertical and horizontal line customization */

QFrame[frameShape=&quot;4&quot;]
{
    color: orange;
}

QFrame[frameShape=&quot;5&quot;]
{
    color: orange;
}

/* QTabWidget elements customization */

QTabWidget::pane {
/*	border-top: 1px solid orange;*/
	background: white;
}

QTabWidget::tab-bar {
    left: 5px;
}

QTabBar::tab {
	background: white;
    min-width: 120px;
    min-height: 20px;
    padding: 2px;
	border-top-left-radius: 5px;
	border-top-right-radius: 5px;
	margin-top: 6px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
	background: white;
	border-top: 1px solid orange;
	border-right: 1px solid orange;
	border-left: 1px solid orange;
}

QTabBar::tab:selected {
	background: white;
	border-top: 2px solid orange;
	border-right: 2px solid orange;
	border-left: 2px solid orange;
}

QTabBar::tab:!selected {
    margin-bottom: 3px;
}

QTabBar QToolButton{
	background-color: white;
	border: none;
	color: black;
}

QTabBar QToolButton::hover{
	background-color: darkorange;
}

QTabBar QToolButton::pressed{
	background-color: orange;
}

QTabBar QToolButton::!enabled{
	background-color: orange;
}

QWidget#tabFile, QWidget#tabManagement, QWidget#tabTools, QWidget#tabHelp, QWidget#tabFileLayout, QWidget#tabManagementLayout, QWidget#tabToolsLayout, QWidget#tabHelpLayout{
	background: white;
}

QLineEdit{
	border: 1px solid orange;
	min-height: 20px;
}

QPushButton{
	background-color: darkorange;
	border: 5px solid darkorange;
	color: white;
}

QPushButton::hover{
	background-color: orange;
	border: 5px solid orange;
}

QPushButton::pressed{
	background-color: darkorange;
	border: 5px solid darkorange;
}

QPushButton::!enabled{
	background-color: orange;
	border: 5px solid orange;
}

/*
	QScroll Area
*/

QScrollArea{
	border: none;
}

/* QScrollBar Horizontal*/

QScrollBar:horizontal {
    background: lightgray;
    height: 10px;
    margin: 0px 40px 0 0px;
	border-radius: 5px;
}

QScrollBar::handle:horizontal {
    background: orange;
	border-radius: 5px;
    min-width: 20px;
}

QScrollBar::add-line:horizontal {
    background: blue;
    width: 20px;
    subcontrol-position: right;
    subcontrol-origin: margin;
	border-top-right-radius: 5px;
	border-bottom-right-radius: 5px;
}

QScrollBar::sub-line:horizontal {
    background: red;
    width: 20px;
    subcontrol-position: top right;
    subcontrol-origin: margin;
    position: absolute;
    right: 20px;
	border-top-left-radius: 5px;
	border-bottom-left-radius: 5px;
}

QScrollBar::sub-line:horizontal:hover {
    background: darkred;
}

QScrollBar::sub-line:horizontal:pressed {
    background: red;
}

QScrollBar::add-line:horizontal:hover {
    background: darkblue;
}

QScrollBar::add-line:horizontal:pressed {
    background: blue;
}

QScrollBar:left-arrow:horizontal{
	image: url(:/images/scrollbar/slide_left.PNG);
    width: 10px;
    height: 10px;
}

QScrollBar::right-arrow:horizontal {
	image: url(:/images/scrollbar/slide_right.PNG);
    width: 10px;
    height: 10px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}

/* QScrollBar Vertical*/

QScrollBar:vertical {
    background: lightgray;
    width: 10px;
    margin: 0px 0px 40px 0px;
	border-radius: 5px;
}

QScrollBar::handle:vertical {
    background: orange;
	border-radius: 5px;
    min-width: 20px;
}

QScrollBar::add-line:vertical {
    background: blue;
    height: 20px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
	border-bottom-right-radius: 5px;
	border-bottom-left-radius: 5px;
}

QScrollBar::sub-line:vertical {
    background: red;
    height: 20px;
    subcontrol-position: bottom right;
    subcontrol-origin: margin;
    position: absolute;
    bottom: 20px;
	border-top-right-radius: 5px;
	border-top-left-radius: 5px;
}

QScrollBar::sub-line:vertical:hover {
    background: darkred;
}

QScrollBar::sub-line:vertical:pressed {
    background: red;
}

QScrollBar::add-line:vertical:hover {
    background: darkblue;
}

QScrollBar::add-line:vertical:pressed {
    background: blue;
}

QScrollBar:up-arrow:vertical{
	image: url(:/images/scrollbar/slide_up.PNG);
	width: 10px;
    height: 10px;
}

 QScrollBar::down-arrow:vertical{
	image: url(:/images/scrollbar/slide_down.PNG);
	width: 10px;
    height: 10px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}

/*QComboBox*/

QComboBox {
    border: 1px solid orange;
	background: white;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
	background: white;
}

QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: white;
}

QComboBox:on {
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: orange;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox::drop-down:hover {
    background: orange;
}

QComboBox::down-arrow {
    image: url(&quot;:/images/buttons/Down_Arrow.PNG&quot;);
	width: 10px;
	height: 10px;
}

QComboBox::down-arrow:on {
    top: 1px;
    left: 1px;
}

QComboBox QAbstractItemView {
    border: 1px solid orange;
    selection-background-color: orange;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox QAbstractItemView:hover {
	color: orange;
}

/*
 * QToolButton
 */

QToolButton {
	background: transparent;
	border: 1px solid transparent;
	color: black;
}

QToolButton::hover {
	background: darkorange;
	border: 1px solid darkorange;
	color: white;
}

QToolButton::pressed {
	background: orange;
	border: 1px solid orange;
	color: white;
}

/*
 * QTableWidget QListWidget
 */

QTableWidget, QListWidget{
	border: 1px solid orange;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="lblDate">
        <property name="text">
         <string>dd/MM/yyyy</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="cbShowTable">
        <property name="text">
         <string>Show table of students</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblReceiptWithoutDiscountDesc">
        <property name="text">
         <string>Receipt (WITHOUT discounts):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblReceiptWithDiscount">
        <property name="text">
         <string>$ 0.00</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblDateDesc">
        <property name="text">
         <string>Current date:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lblReceiptWithoutDiscount">
        <property name="text">
         <string>$ 0.00</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblReceiptWithDiscountDesc">
        <property name="text">
         <string>Receipt (WITH discounts):</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="btExport">
        <property name="text">
         <string>EXPORT CSV</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSelectMonthNYear">
        <property name="text">
         <string>Custom month/year:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbCustomMonthNYear"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QTableWidget" name="tablePricing">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>117</number>
        </attribute>
        <column>
         <property name="text">
          <string>Student</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Course</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Installment</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Price (with discount)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Price (integral)</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <tabstops>
  <tabstop>cbCustomMonthNYear</tabstop>
  <tabstop>cbShowTable</tabstop>
  <tabstop>tablePricing</tabstop>
  <tabstop>btExport</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
